compiler/rustc_ast_lowering/messages.ftl

```diff
@@ -5,6 +5,7 @@ ast_lowering_abi_specified_multiple_times =
 
 ast_lowering_arbitrary_expression_in_pattern =
     arbitrary expressions aren't allowed in patterns
+    .pattern_from_macro_note = the :expr fragment specifier forces the metavariable's content to be an expression
```

Comment:
```suggestion
    .pattern_from_macro_note = the `expr` fragment specifier forces the metavariable's content to be an expression
```

Reply:
It seems the `invalid fragment specifier` error also uses backticks:

```
error: invalid fragment specifier `r#if`
  --> $DIR/invalid-fragment-specifier.rs:7:6
   |
LL |     ($wrong:r#if) => {};
   |      ^^^^^^^^^^^
   |
   = help: valid fragment specifiers are `ident`, `block`, `stmt`, `expr`, `pat`, `ty`, `lifetime`, `literal`, `path`, `meta`, `tt`, `item` and `vis`
```

so switching to those.